djpyfs/models.py

```diff
@@ -55,6 +55,7 @@ def expired(cls):
         return cls.objects.filter(expires=True, expiration__lte = expiration_lte)
 
     class Meta:
+        app_label = 'djpyfs'
```

Comment:
@nedbat should our app labels be further qualified / more canonical? For example org.edx.djpyfs vs just djpyfs? 

Trying not to impose too much java on our python code.